config.py

```diff
@@ -0,0 +1,30 @@
+class Config:
+    """
+    Use this class to share any default attributes with any subsequent
+    classes that inherit from Config.
+    """
+    DEBUG = False
+    TESTING = False
+    
+    # Only required when using the session object
+    # Generated with secrets.token_urlsafe(16)
+    # You could also use os.urandom(16)
+    SECRET_KEY = "muser_data_app"
```

Comment:
What is this key for? Any credentials used to access hosted systems shouldn't be committed in version control.

Reply:
Comment implemented

Reply:
Did something change? - I still see the credentials above. They should be removed from the committed code and if necessary read it programmatically from an unversioned file. Need information in the README for how to configure this too.